SwiftTweaks/TweakTableCell.swift

```diff
@@ -166,6 +167,17 @@ internal final class TweakTableCell: UITableViewCell {
 			let accessoryFrame = colorControlFrame.union(textFrame).union(disclosureArrowFrame)
 			accessory.bounds = accessoryFrame.integral
 
+		case .string:
+			let textFrame = CGRect(
+				origin: .zero,
+				size: CGSize(
+					width: bounds.width * TweakTableCell.stringTextWidthFraction,
```

Comment:
Hyper-nit: let’s round this to the nearest whole pixel. (I’ll do this in a follow-up diff)